### DISCUSSION TITLE
Can't seem to be able to switch programs with Midi CC

### DISCUSSION THREAD

**Discussion by Potajito** [probonopd/MiniDexed#707] [Q&A]
Hi!
Probably a very dumb question, but I can't seem to be able to change programs with midi CC.
I identified the knob that I want to use a cc 3, so I modified minidexed.ini with
`MIDIButtonCh=3`
But probably I'm missing something and that's not the way to do it?

**Comment by diyelectromusic** [comment]
No, that is all about setting up MIDI button equivalents for the UI buttons.  Are you wanting to use MIDI as the user-interface controls or just recognise MIDI Program Change messages directly?  For MiniDexed to receive MIDI Program Change just requires the following to be enabled (which it is by default):
```
MIDIRXProgramChange=1
```

Then any tone generator listening on the MIDI channel you're using will react to the PC messages.

If you want a complete up/down/prev/next/back/home type set of buttons to navigate the MiniDexed menus over MIDI then that is what the MIDIButtons are all about.

Kevin
